### ISSUE TITLE
Pre-work Checklist: Developer: Anthony Perez

### ISSUE THREAD

**Issue by anthonypz** [hackforla/website#4616]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues in the [prioritized backlog](https://github.com/hackforla/website/projects/7#column-7198257) only with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by ExperimentsInHonesty** [comment]
@anthonypz I see that you closed this issue as unplanned.  Did you decide not to join our team?

**Comment by anthonypz** [comment]
@ExperimentsInHonesty Yes, I realized that I may not have at least 6 months of time commitment since I'm on the job search. I do wish to return to Hack4LA in the future as it's something I've been thinking a lot about.

**Comment by anthonypz** [comment]
I have a blocker. I am not able to assign a first issue from the project board to myself. I also noticed I can't change any of the labels related to this github issue.

It appears that I am a part of the `website` and `engineering-write` teams, but not the `website-write` team. For context, I went through the on-boarding last year but didn't have the time to commit, and now I'm picking up where I left off.

Any assistance is appreciated. Thanks.

**Comment by anthonypz** [comment]
> How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?

Five days, starting from the Tuesday (Jan 2, 2024) on-boarding meeting to the Sunday all-team meeting (Jan 7, 2024). Please note that I opened this checklist last year (paused it due to time constraints) and restarted the checklist last Tuesday.

**Comment by ExperimentsInHonesty** [comment]
@anthonypz You have been added back to the repository and the drive.  Please do another good first issue.

**Comment by anthonypz** [comment]
> 1. Progress: "What is the current status of your project? What have you completed and what is left to do?"

My project is in progress. Only one small change is needed, which shouldn't take too long to complete.
> 2. Blockers: "Difficulties or errors encountered."

None so far.
> 3. Availability: "How much time will you have this week to work on this issue?"

All day today.
> 4. ETA: "When do you expect this issue to be completed?"

By 8 p.m. Friday
> 5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

N/A

**Comment by anthonypz** [comment]
Apologies for my lack of communication. I'm ready to contribute.

**Comment by anthonypz** [comment]
Progress: My PR is complete and waiting for review
Blockers: No blockers currently
Availability: Sat and Sun after noon
ETA: The issue has been completed

**Comment by anthonypz** [comment]
Progress: Just started a medium issue
Blockers: None
Availability: M-F after 4 PM
ETA: Friday March 29th EOD

**Comment by anthonypz** [comment]
Progress: I am reading about GH Actions and figuring out how to trigger them manually
Blockers: None. The biggest challenge will be testing the GH Action to ensure everything works correctly.
Availability: M-F after 4 PM
ETA: Saturday April 20th EOD

**Comment by HackforLABot** [comment]
## Activity Log: anthonypz
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- anthonypz opened: https://github.com/hackforla/website/issues/4616 at 05/08/2023, 08:27 PM PDT
- anthonypz assigned: https://github.com/hackforla/website/issues/4616 at 05/08/2023, 09:02 PM PDT
- anthonypz closed as not planned: https://github.com/hackforla/website/issues/4616#event-9216803856 at 05/10/2023, 06:33 PM PDT
- anthonypz commented on: https://github.com/hackforla/website/issues/4616#issuecomment-1566418722 at 05/28/2023, 07:49 PM PDT
- anthonypz opened: https://github.com/hackforla/website/issues/6027 at 01/02/2024, 07:03 PM PST
- anthonypz reopened: https://github.com/hackforla/website/issues/4616#issuecomment-1566418722 at 01/06/2024, 08:47 PM PST
- anthonypz commented on: https://github.com/hackforla/website/issues/6027#issuecomment-1879938152 at 01/06/2024, 08:52 PM PST
- anthonypz closed as not planned: https://github.com/hackforla/website/issues/6027#event-11410309553 at 01/06/2024, 08:52 PM PST
- anthonypz commented on: https://github.com/hackforla/website/issues/4616#issuecomment-1879956104 at 01/06/2024, 09:46 PM PST
- anthonypz assigned: https://github.com/hackforla/website/issues/5965 at 01/07/2024, 10:42 AM PST
- anthonypz commented on: https://github.com/hackforla/website/issues/5965#issuecomment-1880147885 at 01/07/2024, 11:23 AM PST
- anthonypz commented on: https://github.com/hackforla/website/issues/4616#issuecomment-1880151433 at 01/07/2024, 11:38 AM PST
- anthonypz opened: https://github.com/hackforla/website/pull/6075 at 01/07/2024, 03:07 PM PST
- anthonypz PR was merged: https://github.com/hackforla/website/pull/6075#event-11438644466 at 01/09/2024, 09:29 PM PST
- anthonypz issue closed by PR 6075: https://github.com/hackforla/website/issues/5965#event-11438644588 at 01/09/2024, 09:29 PM PST
- anthonypz assigned: https://github.com/hackforla/website/issues/6049 at 02/29/2024, 12:08 AM PST
- anthonypz commented on: https://github.com/hackforla/website/issues/6049#issuecomment-1970627299 at 02/29/2024, 12:16 AM PST
- anthonypz commented on: https://github.com/hackforla/website/issues/4616#issuecomment-1970649002 at 02/29/2024, 12:31 AM PST
- anthonypz commented on: https://github.com/hackforla/website/issues/4616#issuecomment-1971669218 at 02/29/2024, 09:59 AM PST
- anthonypz opened: https://github.com/hackforla/website/pull/6400 at 02/29/2024, 11:23 AM PST
- anthonypz PR was merged: https://github.com/hackforla/website/pull/6400#event-11975111489 at 02/29/2024, 10:19 PM PST
- anthonypz issue closed by PR 6400: https://github.com/hackforla/website/issues/6049#event-11975111631 at 02/29/2024, 10:19 PM PST
- anthonypz assigned: https://github.com/hackforla/website/issues/6086 at 03/07/2024, 10:12 PM PST
- anthonypz commented on: https://github.com/hackforla/website/issues/6086#issuecomment-1985101293 at 03/07/2024, 10:16 PM PST
- anthonypz commented on: https://github.com/hackforla/website/issues/4616#issuecomment-1986700941 at 03/08/2024, 06:37 PM PST
- anthonypz issue closed as completed: https://github.com/hackforla/website/issues/6086#event-12099253406 at 03/13/2024, 12:52 AM PDT
- anthonypz assigned: https://github.com/hackforla/website/issues/6480 at 03/24/2024, 01:03 PM PDT
- anthonypz commented on: https://github.com/hackforla/website/issues/6480#issuecomment-2016950012 at 03/24/2024, 02:30 PM PDT
- anthonypz commented on: https://github.com/hackforla/website/issues/4616#issuecomment-2016951360 at 03/24/2024, 02:32 PM PDT
- anthonypz opened: https://github.com/hackforla/website/pull/6517 at 03/24/2024, 08:16 PM PDT
- anthonypz commented on: https://github.com/hackforla/website/issues/6480#issuecomment-2017137418 at 03/24/2024, 08:18 PM PDT
- anthonypz commented on: https://github.com/hackforla/website/pull/6517#issuecomment-2023921814 at 03/27/2024, 01:29 PM PDT
- anthonypz PR was merged: https://github.com/hackforla/website/pull/6517#event-12308897815 at 04/01/2024, 02:46 AM PDT
- anthonypz issue closed by PR 6517: https://github.com/hackforla/website/issues/6480#event-12308897971 at 04/01/2024, 02:46 AM PDT
- anthonypz assigned: https://github.com/hackforla/website/issues/5182 at 04/12/2024, 08:25 AM PDT
- anthonypz commented on: https://github.com/hackforla/website/issues/5182#issuecomment-2051996761 at 04/12/2024, 08:37 AM PDT
- anthonypz commented on: https://github.com/hackforla/website/issues/4616#issuecomment-2052003282 at 04/12/2024, 08:41 AM PDT
- anthonypz submitted review: https://github.com/hackforla/website/pull/6652#pullrequestreview-2002482228 at 04/15/2024, 07:43 PM PDT
- anthonypz closed as completed: https://github.com/hackforla/website/issues/4616#event-12480931450 at 04/15/2024, 07:50 PM PDT
- anthonypz opened: https://github.com/hackforla/website/pull/6685 at 04/18/2024, 07:05 PM PDT
- anthonypz submitted review: https://github.com/hackforla/website/pull/6685#pullrequestreview-2010469913 at 04/18/2024, 08:43 PM PDT
- anthonypz commented on: https://github.com/hackforla/website/issues/5182#issuecomment-2071176467 at 04/22/2024, 05:36 PM PDT
- anthonypz commented on: https://github.com/hackforla/website/pull/6685#issuecomment-2075976093 at 04/24/2024, 03:48 PM PDT
- anthonypz commented on: https://github.com/hackforla/website/pull/6685#issuecomment-2081218207 at 04/27/2024, 03:31 PM PDT
- anthonypz commented on: https://github.com/hackforla/website/pull/6685#issuecomment-2081310664 at 04/27/2024, 08:32 PM PDT
- anthonypz PR was merged: https://github.com/hackforla/website/pull/6685#event-12633706400 at 04/27/2024, 08:57 PM PDT
- anthonypz issue closed by PR 6685: https://github.com/hackforla/website/issues/5182#event-12633706424 at 04/27/2024, 08:57 PM PDT
- anthonypz opened: https://github.com/hackforla/website/pull/6790 at 04/30/2024, 12:13 PM PDT
- anthonypz PR was merged: https://github.com/hackforla/website/pull/6790#event-12664380895 at 04/30/2024, 02:55 PM PDT
